### DISCUSSION TITLE
FRQ: Add -OR- for filtering different qualifiers

### DISCUSSION THREAD

**Discussion by jasell** [community/community#73715] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

As as User I wish to filter my GitHUb project Views by combing several qualifiers.

Example:
![image](https://github.com/community/community/assets/9511987/e6a780d2-d676-4524-8076-58a91cb5a758)

Expect result is to get a view with all Epic-issues and all other issues assigned a value for Epic
Type=Epic OR -no:Epics

Feel free to use any other symbol for OR, like:
Type=Epic | -no:Epics
Type=Epic , -no:Epics
Type=Epic + -no:Epics

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by benganellison** [comment]
I have the same, we are looking to use projects but not being able to have hierarchies could be overcomed if we could filter on issue_type: Epic or Team: Bravo

then we could show all epics and all issues assigned to the team at the same time
